ray-curator/ray_curator/examples/video/video_split_clip_example.py

```diff
@@ -238,10 +255,10 @@ def main(args: argparse.Namespace) -> None:
         help="Batch size for motion score computation.",
     )
     parser.add_argument(
-        "--motion-score-gpus-per-worker",
```

Comment:
Can we keep this parameter the same and internally we can figure out the amount of memory. This is for the users for cosmos curate moving to Nemo Curator.

Reply:
Also 0 for GPU still works. So we can keep the line Set to 0 to use CPU instead of GPU__